src/sentry/migrations/0050_fix_auditlog_pickled_data.py

```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.28 on 2020-03-06 00:49
+from __future__ import unicode_literals
+
+from bitfield.types import BitHandler
+from django.db import migrations
+
+from sentry.utils.query import RangeQuerySetWrapperWithProgressBar
+
+
+def cleanup_audit_log_data(apps, schema_editor):
+    """
+    Fix `AuditLogEntry` rows that have pickled `Team` models in their `data` field. Also
+    fixes some rows where flags were serialized as a BitHandler rather than a int.
+    See sentry.api.serializers.models.auditlogentry::fix for details.
+    """
+    AuditLogEntry = apps.get_model("sentry", "AuditLogEntry")
+    for audit_log in RangeQuerySetWrapperWithProgressBar(AuditLogEntry.objects.all()):
+        modified = False
+        if "flags" in audit_log.data and isinstance(audit_log.data["flags"], BitHandler):
+            audit_log.data["flags"] = int(audit_log.data["flags"])
+            modified = True
+        if audit_log.event == 3:  # MEMBER_ACCEPT
+            teams = audit_log.data.get("teams")
+            if teams and hasattr(teams[0], "id"):
+                # We have a team in here rather than just the expected data
+                audit_log.data["teams"] = [team.id for team in teams]
+                audit_log.data["teams_slugs"] = [team.slug for team in teams]
```

Comment:
The current OrganizationMemberTeam  `get_audit_log_data` uses `team_slug` and doesn't include the `teams` key at all. Should we normalize on that format since it is being changed now?

Reply:
This audit log uses `OrganizationMember.get_audit_log_data`, which has both. It's created in `ApiInviteHelper.accept_invite`. Or are you suggesting we change that?

Reply:
My mistake. I didn't see that there both `OrganizationMemberTeam.get_audit_log_data` and `OrganizationMember.get_audit_log_data` had totally different key lists, for the same audit entry type.